### PR TITLE
Partial cleanup

### PR DESCRIPTION
**Pull request by brgcode**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas_vol.primitives.VolBox`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (gonzalocasas) [inline .vscode/settings.json:1]: please remove this oen and add `.vscode` to the gitignore

File: src/compas_vol/microstructures/__init__.py [hunk 2]
```diff
@@ -13,7 +13,17 @@
     TPMS
```
Comment (gonzalocasas) [inline src/compas_vol/microstructures/__init__.py:13]:
```suggestion
    TPMS
    LatticePolar
    TPMSPolar
    Voronoi
```

File: src/compas_vol/microstructures/voronoi.py [hunk 3]
```diff
@@ -68,42 +68,5 @@ def get_distance_numpy(self, x, y, z):
...
-            #print(d)
-        #     if d < 0:
-        #         s += 'x'
-        #     else:
-        #         s += '.'
-        # print(s)
-    # print(m.min(), m.max())
-    plt.imshow(m, cmap='gnuplot2')
-    plt.show()
+        print(len(alldistances), type(alldistances[0]))
```
Comment (gonzalocasas) [inline src/compas_vol/microstructures/voronoi.py:72]: this print is odd... shouldn't it be a return of the function or something?

Comment (brgcode) [reply src/compas_vol/microstructures/voronoi.py:73]: not sure but would leave stuff like this to @clementeremy and @worbit

Comment (worbit) [reply src/compas_vol/microstructures/voronoi.py:73]: this method `get_distance_numpy` is not working properly yet, this print was still from the development / debugging phase. can we hide it somehow (set private) until it runs and does what it's supposed to do?

File: src/compas_vol/primitives/vplane.py [hunk 4]
```diff
@@ -62,25 +60,25 @@ def get_distance_numpy(self, x, y, z):
...
+
+#     # for y in range(-15, 15):
+#     #     s = ''
+#     #     for x in range(-30, 30):
+#     #         d = p.get_distance(Point(x * 0.5, 0, -y))
+#     #         if d < 0:
+#     #             s += 'x'
+#     #         else:
+#     #             s += '.'
+#     #     print(s)
```
Comment (worbit) [inline src/compas_vol/primitives/vplane.py:84]: that's good, i think eventually all the commented main sections can be removed. some of them are already (and others maybe will be) moved to the examples folder. i used them for debugging / testing mainly

**Review by worbit — APPROVED** [review]
LGTM, so LGTM

**Pull request merged**